docs/DeveloperGuide.md

```diff
@@ -571,13 +573,14 @@ The below sequence diagram displays the interactions while executing the command
 
 ### Summary Statistics Feature
 
+#### Implementation
+
 This is a new command to view summary statistics of all students.
-Entering the command `summary` results in a popup window in the gui. The popup window consists of summary data including total
+Entering the command `summary` results in a popup window in the GUI. The popup window consists of summary data including total
 number of students, mean grade, and standard deviation of grades. Additionally, a colored pie chart is displayed of the
 students' grade distribution.
 
-Implementation:
-The `summary` command is implemented as such: 
+Given below is an example usage scenario:
 
 - `LogicManager`'s execute method calls the `parseCommand` method from `AddressBookParser`
 - `parseCommand` creates a `SummaryCommand`
```

Comment:
Slightly nitpicky, but can you change these to Step 1., Step 2., etc. to fit the rest of the features?

Reply:
Done in latest commit